ledger-core/virtual/execute/execute.go

```diff
@@ -124,6 +124,7 @@ func (s *SMExecute) prepareExecution(ctx context.Context) {
 }
 
 func (s *SMExecute) migrationDefault(ctx smachine.MigrationContext) smachine.StateUpdate {
+	ctx.Log().Trace("migrationDefault")
```

Comment:
Do you really need? Wouldn't it be written by Conveyor's step logger?

Reply:
removed